app/src/main/java/com/example/riden/models/User.java

```diff
@@ -1,12 +1,18 @@
 package com.example.riden.models;
 
 import com.parse.ParseClassName;
+import com.parse.ParseFile;
 import com.parse.ParseObject;
+import com.parse.ParseUser;
 
-@ParseClassName("User")
-public class User extends ParseObject {
+@ParseClassName("_User")
+public class User extends ParseUser {
     public static final String USERNAME_KEY = "username";
     public static final String PHONENUMBER_KEY = "phoneNumber";
```

Comment:
nits: can we change this to PHONE_NUMBER_KEY i.e. use underscores for new word separation